packages/devextreme/js/__internal/core/widget/widget.ts

```diff
@@ -265,8 +266,9 @@ class Widget<
     return this._getActiveElement();
   }
 
-  _isFocusTarget(element: Element): boolean {
+  _isFocusTarget(element: Element | undefined): boolean {
```

Comment:
Suggestion: we can use shorter form
```suggestion
  _isFocusTarget(element?: Element): boolean {
```

Reply:
I'll consider in the further PRs